packages/sanes-browser-extension/src/extension/background/model/persona/config/codec.ts

```diff
@@ -59,14 +59,17 @@ export function chainConnector(
   nodeUrl: string,
   scraper: string | undefined,
 ): ChainConnector {
+  const expectedBnsChainId: ChainId | undefined = undefined;
+  const expectedLiskChainId: ChainId | undefined = undefined;
+  const expectedEthereumChainId: ChainId | undefined = undefined;
```

Comment:
Why those constants is here?

Reply:
The updated chain connectors allow us to specify a chain ID, which we expect from the blockchain node. This prevents some basic misconfiguration on the blockchain side.

Recently I connected to testnet.lisk.io but got an answer from a mainnet node due to misconfiguration as Lisk side.

In https://github.com/iov-one/ponferrada/issues/578 we will fill the variables.